adapters/tlsio_openssl.c

```diff
@@ -864,7 +870,7 @@ static int load_cert_crl_http(
 
     OCSP_set_max_response_length(rctx, 1024 * 1024);
 
-    if (!OCSP_REQ_CTX_http(rctx, "GET", path))
+    if (!OCSP_REQ_CTX_http(rctx, "GET", url)) // path
```

Comment:
could you please double check the "Host" field in the http header contains correct host name? Not sure whether this function will take host from url or from BIO_new_connect().